### PR TITLE
Added geoalchemy2

### PR DESCRIPTION
**Pull request by agyoungs**
This python module appears to be a debian binary in python 3, but not python 2

### CODE REVIEW

**Review by cottsay — CHANGES_REQUESTED** [review]
Hi! Please see the [CONTRIBUTING.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-rosdep-rules) and update this change based on those guidelines. Off the top of my head, I can see that the key should end in `-pip`, and a link to the PyPI page in the change description would be appreciated.

Thanks!

**Comment by agyoungs** [comment]
Just added `-pip` to the end of the tag.

[geoalchemy2](https://pypi.org/project/GeoAlchemy2/)

I needed this in a package that I'm working on that uses a SpatiaLite extended SQLite database. This library works with SQLAlchemy which is already included as a rosdep. It looks like it is available in 19.04 as a debian package, but only for python3 at the moment.

Thanks!

**Review by agyoungs — COMMENTED** [review]
Addressed changes from review (i.e. adding `-pip` to the end)

File: rosdep/python.yaml [hunk 1]
```diff
@@ -1573,6 +1573,13 @@ python-genshi:
   fedora: [python-genshi]
   gentoo: [dev-python/genshi]
   ubuntu: [python-genshi]
+python-geoalchemy2-pip:
+  debian:
+    pip:
+      packages: [geoalchemy2]
+  ubuntu:
```
Comment (clalancette) [inline rosdep/python.yaml]: If we're doing a `-pip` package, we may as well add support for fedora and osx as well (like https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L157).  Thanks.

**Review by clalancette — CHANGES_REQUESTED** [review]
Small changes to the key.

**Comment by clalancette** [comment]
No response for almost a month, so I'm closing this.  If you'd still like to get this key in, feel free to reopen this PR and make the changes requested.  Thanks for the contribution.

**Pull request closed without merging**